CMakeLists.txt

```diff
@@ -664,14 +654,17 @@ if(JPSFIRE)
 endif()
 
 # todo make this system
-add_library(thirdParty STATIC ${THIRD_PARTY_SRC})
```

Comment:
I don't think it is necessary to change to dynamic linking

Reply:
That was not intentional. `Static` should do.